pos_customer_display/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,11 @@
+This module adds support for LED Customer Display in the Point of Sale.
+
+It has been tested with the following devices:
+
+* Bixolon BCD-2000 (https://www.bixolon.com/product_view.php?idx=162),
```

Comment:
Hi. did you tested with that device ? if yes, you could remove the roadmap part.

Reply:
I did test with BCD-2000 (I have one with me at the moment, cf the picture I sent on twitter), but it was configured in BCD-2000K mode with 2 x 20. So the roadmap entry is still valid; we could still improve the code to take advantage of 4x30. But I don't plan to implement that myself. For me, the priority is to improve/cleanup the current JS code, with the help of some JS devs.